docs/data/data-grid/aggregation/aggregation.md

```diff
@@ -2,7 +2,7 @@
 title: Data Grid - Aggregation
 ---
 
-# Data Grid - Aggregation [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-pro/)
+# Data Grid - Aggregation [<span class="plan-premium"></span>](https://mui.com/store/items/mui-x-premium/)
```

Comment:
@joserodolfofreitas This wasn't caught by ahrefs 🙃, the don't crawl out of scopes URLs (same domain), but they do crawl external domains for 301 🤦‍♂️